services/api/src/clients/harborClient.js

```diff
@@ -12,10 +12,13 @@ const lagoonHarborRoute = R.compose(
   R.propOr('', 'LAGOON_ROUTES'),
 )(process.env);
 
+const apiVersion = R.propOr('v1', 'HARBOR_API_VERSION', process.env);
```

Comment:
doesn't this mean that if `HARBOR_API_VERSION` is not set the default will be `v1`?
let's make the default `v2` :)

Reply:
So, we're currently only setting that variable if we deploy the API via Helm. If we want this to be backwards compatible, we would either need to pass that var into this container somehow or leave `v1` as the default.

Reply:
I think we're talking about different backwards compatibilities then:
- With Lagoon 2.0 (which is the main branch) we will only be able to deploy lagoon/api via helm, there will be no other way to do this (deploying lagoon via lagoon will not be possible anymore).
- But we might have customers that deploy lagoon/api via helm and want to use an Harbor Version 1 , but that should not be the standard anymore, as we deploy Harbor 2 via our Harbor charts in lagoon-charts repo.
So we can make everything Harbor 2 by default, just it would be cool if we can still support Harbor 1 for some Lagoon installations.

Reply:
Ah yeah, I was trying to keep this usable for a Lagoon release which would be deployed via Lagoon. Your points are salient; I'll push that change up shortly. 😸